### PR TITLE
hw jdbc

### PR DESCRIPTION
**Pull request by Taniann**

### CODE REVIEW

File: src/main/java/ua/mateacademy/tania/lesson17/jdbc/OfficeDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            preparedStatement.setString(1, region);
+            ResultSet rs = preparedStatement.executeQuery();
+            while (rs.next()) {
+                offices.add(extraxtFromResultSet(rs));
+            }
+        }
+        return offices;
+    }
+
+    private Office extraxtFromResultSet(ResultSet rs) throws SQLException {
```
Comment (Daniluko) [inline src/main/java/ua/mateacademy/tania/lesson17/jdbc/OfficeDaoImpl.java:48]: Good

**Review by Daniluko — APPROVED** [review]
Looks good

File: src/main/java/ua/mateacademy/tania/lesson17/jdbc/OfficeDaoImpl.java (hunk 1 above)
Comment (SerhiiPasko) [reply src/main/java/ua/mateacademy/tania/lesson17/jdbc/OfficeDaoImpl.java:48]: It isn't resultSet it is line in it

File: src/test/java/ua/mateacademy/tania/lesson17/jdbc/OrderDaoImplTest.java [hunk 2]
```diff
@@ -26,4 +27,20 @@ public void testFindOrderByIdNotPresent() throws SQLException {
         Order order = orderDao.findOrderById(NOT_EXIST_ORDER);
         assertNull(order);
     }
+
+    @Test
+    public void testGetAllOrders() throws SQLException {
```
Comment (SerhiiPasko) [inline src/test/java/ua/mateacademy/tania/lesson17/jdbc/OrderDaoImplTest.java:32]: Also good test cheak both results (must be same) from two different methods

**Review by SerhiiPasko — APPROVED** [review]
Good. Read comments

**Pull request merged**